### PR TITLE
[Basic UI] Add support for webaudio

### PR DESCRIPTION
**Pull request by wborn**
Similar to https://github.com/openhab/openhab-webui/pull/1422 but for Basic UI. :slightly_smiling_face:

### CODE REVIEW

**Comment by lolodomo** [comment]
I agree with the need of an option.

**Comment by wborn** [comment]
Config option added. :heavy_check_mark:

**Comment by lolodomo** [comment]
I will test and review.

**Comment by wborn** [comment]
@lolodomo if the changes don't work, make sure you have:

* Reloaded Basic UI after enabling the Basic UI Web Audio setting
* Navigated to a sitemap
* Interacted with the Basic UI (or change your browser setting to always allow audio playback for the site)

File: bundles/org.openhab.ui.basic/src/main/resources/OH-INF/config/config.xml
Comment (lolodomo) [inline bundles/org.openhab.ui.basic/src/main/resources/OH-INF/config/config.xml:45]: You used the same pattern (text type) as for other parameters.
I do not understand why boolean type was not used for enableIcons, condensedLayout, ... parameters.
But your new parameter is at least coherent with others.

Comment (wborn) [reply bundles/org.openhab.ui.basic/src/main/resources/OH-INF/config/config.xml:45]: Yes I kept the new parameter coherent with the other config parameters. It seems that @kaikreuzer added the inital config parameter in https://github.com/eclipse-archived/smarthome/pull/540 like this. Maybe because it easier to understand (compared to a checkbox)?

**Review by lolodomo — APPROVED** [review]
LGTM. I will merge after a successful test.

**Comment by lolodomo** [comment]
Tested and working with barking.mp3

**Comment by lolodomo** [comment]
Thank you @wborn

**Comment by wborn** [comment]
Maybe you can add the "enhancement" label to this PR so this feature ends up in the final release notes @openhab/webui-maintainers ?

**Pull request merged**
